app/models.py

```diff
@@ -39,11 +40,20 @@ def get_file_path(self):
             return file_path
 
     # todo: add to celery task
-    def generate_wrapped_file(self, upload_file):
-        '''
-        Return generate file with name of episode prefix from upload_file
-        '''
-        pass
+    def generate_wrapped_file(self, episode_path, name_path):
```

Comment:
Would it be better to create generate_wrapped_file functions in a separate file? Not in the Model.